frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -55,7 +55,8 @@ export interface MultiplexResult {
   disease: {
     name: string;
   };
-  testResult: TestResult;
+  testResult?: TestResult;
+  result?: TestResult;
```

Comment:
Why do we need a new field with the same type but different name?

Reply:
See [this discussion](https://github.com/CDCgov/prime-simplereport/pull/3920#discussion_r911505600) for details, but the short version is that the patient experience backend and graphQL endpoints return _slightly_ different fields, and we need to allow both.